README.md

```diff
@@ -10,15 +10,43 @@ The purpose of this repository is to allow easy management of all Pop!\_OS relat
 
 Binary packages are hosted [on Launchpad](https://launchpad.net/~system76/+archive/ubuntu/pop/+packages). Many packages have source on Github, under the [Pop!\_OS organization](https://github.com/pop-os).
 
+## Developer Resources
+For instructions on how to build the shell:
+	* [Cosmic, our current gnome-based shell](https://github.com/pop-os/cosmic). 
+	* [Cosmic-Epoch, the new Rust based shell](https://github.com/pop-os/cosmic-epoch).
+
 To view a list of all included repositories, see [REPOS.md](./REPOS.md)
+Developer chat: https://chat.pop-os.org/
```

Comment:
This line break did not work. Markdown requires an extra line break in between these two lines or four spaces at the end of the previous line in order to actually break the line.

The `list of all included repositories` line should probably not be moved down into this section. It could be moved higher than the current position into the very first section.

The `Developer chat:` line does not have any context. That Mattermost server is for users too, not just developers. The ~development channel is specifically to discuss development, and can be linked to directly like this: https://chat.pop-os.org/pop-os/channels/development